05-setting-up.Rmd

```diff
@@ -9,35 +9,163 @@ ottrpal::set_knitr_image_path()
 ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g21a84b32106_0_13")
 ```
 
+## Understand why project organization is key to reproducible analyses
 
+Keeping your files organized is a skill that has a high long-term payoff. As you are in the thick of an analysis, you may underestimate how many files and terms you have floating around. But a short time later, you may return to your files and realize your organization was not as clear as you hoped.  
 
-## Understand why project organization is key to reproducible analyses
+@Tayo2019 discusses four particular reasons why it is important to organize your project:
+
+> 1. Organization increases productivity. If a project is well organized, with everything placed in one directory, it makes it easier to avoid wasting time searching for project files such as datasets, codes, output files, and so on.
+> 2. A well-organized project helps you to keep and maintain a record of your ongoing and completed data science projects.
+> 3. Completed data science projects could be used for building future models. If you have to solve a similar problem in the future, you can use the same code with slight modifications.
+> 4. A well-organized project can easily be understood by other data science professionals when shared on platforms such as Github.
+
+Organization is yet another aspect of reproducibility that saves you and your colleagues time!
 
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Major point!! example image"}
 ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g2fea8805c08_0_1121")
 ```
 
-## Understand general principles of project organization
+## General principles of project organization
+
+Project organization should work for you and not the other way around. Maintainably organized should be the goal. Something that is effective but can be maintained long term. As you can imagine, organizational schemes are not one size fits all.
```

Comment:
```suggestion
Project organization should work for you and not the other way around. The goal should be organization that is maintainable long term.  As you can imagine, organizational schemes are not one size fits all.
```

Reply:
hmmm since we say one size fits all thing right after I also changed this slightly to use different phrasing